Dockerfile

```diff
@@ -1,3 +1,48 @@
-FROM scratch
-CMD eksctl
-COPY --from=weaveworks/eksctl-builder:latest /out /
+# Make sure to bump the version of EKSCTL_DEPENDENCIES_IMAGE if you make any changes
+# to this file
```

Comment:
Will need to update this and other files. Now you need to run `make update-build-image`, we might want to add add `make publish-build-image` as well.